### DISCUSSION TITLE
where condition in findMany: value of a field equals any value in the specified array

### DISCUSSION THREAD

**Discussion by dakshshah96** [prisma/prisma#7628] [Q&A] [answered]
I have a `status` column of type `TEXT` in my Postgres table named `tasks`. This column can contain one of these values: `pending`, `open`, `completed`.

I want the consumers of my GraphQL API to be able to pass a value or multiple values in an array for `status` and return the appropriate results. The GraphQL schema looks like this:

```
type Task {
  id: Int!
  status: String!
  description: String!
}

type Query {
  tasks(
    status: [String]!
  ): Task
}
```

If I want to filter tasks by a specific value of `status`, I can do this (assuming `status` argument is of type string):

```
prisma.task.findMany({
  where: {
    status: args.status
  }
})
```

How do I filter for tasks such that the value of the `status` column equals to any value in the specified array? For example, if I passed `['pending', 'completed']` for `status` in the GraphQL query, I would get tasks whose `status` is either `pending` or `completed` and the tasks with status of `open` will be excluded.

**Comment by dakshshah96** [comment, marked as answer]
My bad, found the answer: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#in

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
